ui/dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM nginx:stable
+
+RUN rm /etc/nginx/conf.d/default.conf
+
+COPY nginx.conf /etc/nginx/conf.d
+
+COPY dist /usr/share/nginx/html
```

Comment:
Are you going to build the static files outside the Docker and then `COPY` it to the image?

I think the static files should be built inside the Dockerfile for the following:

1. Consistency and Reproducibility
2. Dependency Management: Manages all build dependencies within the Docker image, avoiding contamination of the host machine (GitHub Actions).
3. Security: Enhances security by providing transparency in the build process.

For example, in our backends, we don't build it and then copy the binary to the image.

https://github.com/reearth/reearth-visualizer/blob/2f0c3a8a3003ec6c0243a3d451edbab84ac7d942/server/Dockerfile#L15

Reply:
Makes sense, Done.
Ref: https://github.com/reearth/reearth-flow/pull/243/commits/f94eaebe1ec1828d652adda55ed3fab4f4b8a6e1